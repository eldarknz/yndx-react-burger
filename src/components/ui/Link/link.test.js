import React from "react";
import renderer from "react-test-renderer";
import { shallow } from 'enzyme';

import FancyLink from "./Link";

/*it("The link is rendering without errors", () => {
    const component = renderer.create(<FancyLink>Ссылка</FancyLink>).toJSON();
    expect(component).toMatchSnapshot();
});*/

/*it('Нажатие на кнопку вызывает корректный alert', () => {
    window.alert = jest.fn();

        // Рендерим ссылку в переменную
    const output = shallow(
        <FancyLink onClick={() => alert('Ссылка')}>Ссылка</FancyLink>
    );
        // Имитируем нажатие на ссылку
    output.simulate('click');
        
        // Проверяем, что alert сработал с правильным текстом предупреждения
    expect(window.alert).toHaveBeenCalledWith('Ссылка');
});*/