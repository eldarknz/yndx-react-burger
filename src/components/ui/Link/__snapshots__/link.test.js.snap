// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The link is rendering without errors 1`] = `
<div
  className="link"
>
  Ссылка
</div>
`;
